website-deep-dive.mdx

```diff
@@ -0,0 +1,145 @@
+
+<ArticleVideo
+  platform="html"
+  src="https://us-east-1.storage.xata.sh/rp7qvd6get3vvd2hmjfqhuf0a8"
+  width="1604"
+  height="1080"
+/>
+
+### Color modes
+
+To refresh our brand we did a few adjustments to our color tokens. This had a ripple effect, influencing both our documentation and app. Additionally, we made a bold choice to exclusively design in dark mode, a decision we elaborated on in the "Navigating challenges" section.
+
+However, amidst these changes, we collectively decided to maintain two color modes for our documentation and app, prioritizing an exceptional Developer Experience (DX). This thoughtful decision ensures that our users have the freedom to select the color mode that aligns best with their preferences, thus elevating their overall journey with our platform.
+
+![Documentation color mode](/images/website-deep-dive/documentation-color-mode@2x.png)
+
+### Illustrations
+
+We also introduced new illustrations to our website, which we created using Figma. For each feature page, we designed unique hero illustration to represent the feature.
+We opted for a minimalist approach to ensure that the text remains the primary focus.
+
+![Feature pages hero images](/images/website-deep-dive/fp-hero-images@2x.png)
+
+We also created a set of illustrations to represent the different features of our platform.
+
+![Feature pages images](/images/website-deep-dive/fp-illustrations@2x.png)
+
+### Embracing fun and welcoming vibes
+
+As we explained earlier, one of the things we were eager to bring with our brand was "fun." We wanted to break away from seriousness a bit. And since our logo is a butterfly, we wanted it to somehow be present on our website and brand. We wanted to ensure our users felt warmly welcomed and included. That's why we decided to add a bit of fun into the mix without going overboard.
+
+When scrolling the landing page, you can find some small animations. An example of this is when hovering over the "start free" button, the butterfly emits some hearts. This is because we want to show our users that we're happy they're signing up for our product.
+
+<ArticleVideo
+  platform="html"
+  src="https://us-east-1.storage.xata.sh/5nu77upshl6ql2ua515kmlqim8"
+  width="3852"
+  height="1080"
+/>
+
+## Implementing the technical aspects of the redesign
+
+Our website is part of a monorepo where apps are developed using Next.js, a decision we've elaborated on in another post: [Using Next.js to improve speed and efficiency at Xata](/blog/nextjs-to-improve-speed-and-efficiency).
+
+As we implemented, we took certain considerations into account. If a component needed to be used across the website, docs, and app, it would be included in our [Design System](https://design.xata.io/). For instance, the logos. If the component is solely for the website, then it remains within the website scope.
```

Comment:
```suggestion
As we implemented the website, we took certain considerations into account. If a component needed to be used across the website, docs, and app, it would be included in our ![Design System](https://design.xata.io/). For instance, the logos. If the component is solely for the website, then it remains within the website scope.
```